chaoslib/openebs/cstor_pool_failure.yaml

```diff
@@ -0,0 +1,73 @@
+- name: Derive PV from application PVC 
```

Comment:
This is a good util to check volume status/sanity before proceeding to the chaos (pool deployment delete). What do you think about renaming this to cstor_volume_sanity.yaml - we could include this as a pre-chaos step in every cstor test

Reply:
The cstor pool delete could then be called separately from the test? i.e., the chaosutil selection jinja template could call this instead.

Reply:
I made this as cstor_verify_pool_provisioning.yml, and, included this in cstor_pool_failure.yml.
This file cstor_verify_pool_provisioning.yml can be moved to different place to use this from other directories.